tools/bump-version

```diff
@@ -0,0 +1,105 @@
+
+  const date = new Date()
+  const year = date.getFullYear()
+  const month = date.getMonth() + 1
+
+  const versionNameCounter = year === versionNameParts[0] && month === versionNameParts[1] ? versionNameParts[2] + 1 : 0
+  const newVersionName = `${year}.${month}.${versionNameCounter}`
+  const newVersionCode = versionCode ? versionCode + 1 : undefined
+
+  console.debug(`New version name: ${newVersionName}`)
+  if (newVersionCode) {
+    console.debug(`New version name: ${newVersionCode}`)
+  }
+
+  return {
+    versionName: newVersionName,
+    versionCode: newVersionCode
+  }
+}
+
+const commitVersionBump = async (newVersion, { versionFile, deliverinoPrivateKey, owner, repo, branch }) => {
+  const appId = 59249 // https://github.com/apps/deliverino
+  const privateKey = Buffer.from(deliverinoPrivateKey, 'base64').toString('ascii')
+
+  const octokit = new Octokit({ authStrategy: createAppAuth, auth: { id: appId, privateKey: privateKey } })
+  const { data: { id: installationId } } = await octokit.apps.getRepoInstallation({ owner, repo })
+  const { data: { token: installationToken } } = await octokit.apps.createInstallationToken({ installationId })
+
+  const appOctokit = new Octokit({ auth: installationToken })
+  const versionFileContent = await appOctokit.repos.getContents({ owner, repo, path: versionFile, ref: branch })
+
+  const { versionCode, versionName } = newVersion
+  const contentBase64 = Buffer.from(JSON.stringify(newVersion)).toString('base64')
+
+  const message = versionCode
+    ? `Bump version name to ${versionName} and version code to ${versionCode}\n[skip ci]`
+    : `Bump version name to ${versionName}\n[skip ci]`
+
+  await appOctokit.repos.createOrUpdateFile({
+    owner,
+    repo,
+    path: versionFile,
+    content: contentBase64,
+    branch,
+    message,
+    sha: versionFileContent.data.sha
+  })
+
+  console.debug(`Bumped version successfully commited with message "${message}"`)
+}
+
+program
+  .command('bump-version')
+  .description('bump the current versionName (and versionCode if available) and commit to github')
+  .action(async () => {
+    try {
+      const { dryRun, versionFile, deliverinoPrivateKey, owner, repo, branch } = program
+      console.log('versionFile', versionFile)
+      if (!dryRun) {
+        if (!deliverinoPrivateKey) {
+          throw new Error('deliverino private key not specified')
+        } else if (!owner) {
+          throw new Error('owner not specified')
+        } else if (!repo) {
+          throw new Error('repo not specified')
+        } else if (!branch) {
+          throw new Error('branch not specified')
+        }
+      }
+
+      const newVersion = calculateNewVersion(versionFile)
+
+      if (!dryRun) {
+        await commitVersionBump(newVersion, { versionFile, deliverinoPrivateKey, owner, repo, branch })
+      }
+
+      return JSON.stringify(newVersion)
+    } catch (e) {
+      console.error(e)
+      process.exit(1)
```

Comment:
This `exit` can be removed as it is the last line of this script.

Reply:
No, because it is needed to return an error status code. Otherwise, circleCI will proceed delivering even if bump version failed.